electron_app/src/preload.js

```diff
@@ -14,8 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const { ipcRenderer } = require('electron');
+const { remote, ipcRenderer, webFrame } = require('electron');
+const SpellChecker = remote.require('spellchecker');
 
 // expose ipcRenderer to the renderer process
 window.ipcRenderer = ipcRenderer;
+webFrame.setSpellCheckProvider('en-US', true, {
```

Comment:
Uhm why hardcoded to en-US? Riot is used with many different languages and not just english

Reply:
I think this parameter is basically useless, it is provided to electron and not the spellchecker module, so I didn't see any logical way it could actually do anything. The spellchecking is done by a call to SpellChecker.isMisspelled(text), which uses native spellchecking APIs or hunspell.

Reply:
Reading thru electron source it looks like it may change the behavior of how it deals with contractions in different languages, I will update this PR accordingly.